src/components/Dropdown/props.ts

```diff
@@ -118,4 +118,19 @@ export type DropdownProps = {
    * list of triggers that can open the Dropdown (accepts: click, hover, contextMenu).
    */
   triggers?: DropdownTrigger[],
+
+  /**
+   * Whether the dropdown menu is currently open.
+   */
+  open?: boolean,
+
+  /**
+   * Called when the open state changes. Not triggered when hidden by click item.
+   *
+   * @param open the current open state of the Dropdown
+   * @param info an object with the following props:
+   * - source: the source of the open change, can be 'trigger' or 'menu'
+   * @returns
+   */
+    onOpenChange?: (open: boolean, info: {source: 'trigger' | 'menu'}) => void
```

Comment:
define an enum for mapping the values

Reply:
It would make it incompatible with the underlying type of Antd, which is defined exactly as I did here

Reply:
I could define it with a type instead? Such as:
```typescript
export type OpenChangeInfoSource = 'trigger' | 'menu'
```
This would keep the compatibility

Reply:
better using enum for developer experience when using the component. I'd expect to use something like `Dropdown.OpenChangeSource.Trigger` and `Dropdown.Placement.Top`

Reply:
defined an enum and a function to remap it to the underlying antd function correctly